templates/components/itilobject/timeline/form_validation.html.twig

```diff
@@ -204,6 +204,7 @@
                     {{ fields.field(
                         '_add_validation',
                         subitem.dropdownValidator({
+                            'parent': item,
```

Comment:
Same modification should proably be made on `templates/components/itilobject/fields/global_validation.html.twig`.

Reply:
Remark is still relevant.